README.md

```diff
@@ -135,6 +135,22 @@ const didWithCap = did.withCapability(cacao)
 const didWithCap2 = new DID({provider, resolver: KeyResolver.getResolver(), capability: cacao})
 ```
 
+## Security Considerations
+
+Ceramic allows for keys attached to DIDs to continue making updates to streams after revocation for a certain grace period. This is done to ensure that if multiple updates were committed to different nodes before the key was revoked, but the nodes chose to anchor commits at different times, the updates still make it through.
+
+1. Commit for an update is made to Node A
+2. Key revocation commit is made to Node B
+3. Node A later tries to anchor the commit made earlier
+
+In this case, an external outlook might lead you to believe Node A's commit is invalid because the key had been revoked, but that's not necessarily true as in this case the commit was made earlier, just not anchored at that time. For this reason, a grace period is provided for updates created by revoked keys to still be verified successfully.
+
+Specifically, the `verifyJWS` function in `dids` uses `options.revocationPhaseOutSecs` as the seconds representing the grace period within which signatures authored by a given key will still be considered valid.
```

Comment:
```suggestion
Specifically, the `verifyJWS` function in `dids` uses `options.revocationPhaseOutSecs` as the seconds representing the grace period within which signatures authored by a given key that otherwise appears to be revoked will still be considered valid.
```